### PR TITLE
feat(*): update documentation link globally

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/sprint-2302-2304`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-10666
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
Helpcenter GA - Migration to snow: Updation documentation link globally across manager.

### CODE REVIEW

**Review by ghyenne — DISMISSED** [review]

File: packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/constants.ts [hunk 1]
```diff
@@ -30,6 +32,8 @@ const consts: UsefulLinkConstants = {
       TN: `${helpRoot}/fr-tn${homeIndex}`,
     },
     tasks: 'https://www.status-ovhcloud.com/',
+    supportTicketUrl: 'https://help.ovhcloud.com/csm?id=contact_us_ovh',
+    viewSupportTicketsUrl: 'https://help.ovhcloud.com/csm?id=csm_cases_requests',
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/constants.ts:36]: **nitpick:** No need to add both. 
/dedicated/support/tickets/new -> This is the URL for Support page in EU and CA
/dedicated/ticket -> This is the URL for Support ticket page in US

So, the above links must be added based on the region.

**Review by anooparveti — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Comment by anooparveti** [comment]
Hi @sachinrameshn,
I see that the PR is closed. If the branch is not required anymore, may I request you to delete the branch on origin?
Thanks!

**Comment by sachinrameshn** [comment]
> Hi @sachinrameshn, I see that the PR is closed. If the branch is not required anymore, may I request you to delete the branch on origin? Thanks!

Its needed it will be re looked in the month of September, Thanks !!

**Comment by sachinrameshn** [comment]
Added a comment on ticket, to check if we can keep it open or not.

**Comment by sachinrameshn** [comment]
Can be closed confirmed by Marion.

**Pull request closed without merging**